### ISSUE TITLE
Load jhu-csse backfilled data

### ISSUE THREAD

**Issue by huisaddison** [cmu-delphi/covidcast-indicators#23]
Currently, the `jhu-csse` pipeline derives its daily incidence and cumulative by reading from the `confirmed_US` and `deaths_US` sheets here: 
https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series

(Note that daily reports are also available, but they do not stretch very far in time for the United States).

I recently learned that JHU backfills its US data: https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_time_series/Errata.csv

The number of changes to the US data are sufficiently small that ignoring them probably would not be much worse than rounding error; but in principle we should probably incorporate them.

**Comment by nickreich** [comment]
Hi all - I'd like to mention that this issue remains a major factor in our project at the COVID Forecast Hub not being able to more fully incorporate covidcast into our everyday pipeline. We have had to essentially duplicate covidcast functionality to provide our own versions of the JHU-CSSE dataset. This is crazy because you all have such polished functionality for this, but in order to make the switch to adopting covidcast, we need full backfilled data for jhu-csse to be available. Any updates on when this might be solved?

**Comment by krivard** [comment]
We're currently repairing major issues in the JHU indicator. Once we have basic JHU functionality back online, we'll be able to pursue this item with greater focus. 

In the meantime, work on integrating backfill is proceeding in parallel. Unfortunately, as you are probably aware, JHU has changed its geo coding several times since the beginning of the pandemic, and the git history includes several errors introduced by hand-edits. Both factors make it difficult for an automatic exporter to parse the data history. If your team has solved any these problems, we'd be eager to swap notes -- let me know.

**Comment by nickreich** [comment]
From our perspective, we want the data as JHU had reported it at a given moment in time. we'd want the data exactly as JHU released it, so in some senses we want it NOT cleaned up. we want the data as models would have seen it "as of" a particular time, funny geo-codings and all. 
It strikes me that we have only been using the US data from JHU and I haven't been aware of major changes in the geocoding there. (we haven't run into any issues like this yet that I am aware of.) Maybe starting with the US data would be a place to split this into a few separate tasks?

**Comment by capnrefsmmat** [comment]
The US data is indeed where we've run into geocoding issues with JHU, such as their switch in handling of New York City. We haven't worked with their international data at all yet.

**Comment by nickreich** [comment]
I see what you mean. I think that's just how the data is. it's a feature, not a bug to be corrected. E.g. if we want covidcast to be a faithful representation of what JHU was showing at a given time (that's what we use it for when modeling retrospectively) then you all in the DELPHI group shouldn't be changing the FIPS codes that are in the JHU data. you are just mirroring what they had. Is it perfect, no? should you maybe make a separate "clean" JHU signal that tries to address these issues, maybe? but as a first pass, just mirroring this imperfect data would be very helpful for us at the COVID Forecast Hub (and for all the modeling teams that are contributing)!

**Comment by krivard** [comment]
> it's a feature, not a bug to be corrected. 

The kinds of errors I'm talking about are the accidental deletion of commas and other formatting errors that invalidate the whole file. Not sure what the right call is there other than skipping those versions entirely.

**Comment by nickreich** [comment]
That seems reasonable to me, to just skip the version entirely.

**Comment by nickreich** [comment]
but the changes in geocoding issues should not block this from being updated, imho.

**Comment by chinandrew** [comment]
@eujing, @dshemetov , and I looked into this yesterday. Looks like the files which were available on each day by 10AM ET could all be parsed and did not contain nan values, so we can probably backfill them without issues. Working with @eujing right now on that.

**Comment by krivard** [comment]
We have a dataset that correctly configures issues from April 2 to October 28. I have run it successfully on staging, and will be running it in production on Monday to avoid any nasty weekend surprises.
